app/modules/intelligence/prompts_provider/base_prompts_provider.py

```
@@ -0,0 +1,67 @@
+from types import SimpleNamespace
+from typing import List, Dict, Any
+from abc import ABC
+
+from app.modules.intelligence.prompts.prompt_model import PromptType
+from app.modules.intelligence.prompts_provider.agent_types import AgentRuntimeLLMType
+from app.modules.intelligence.prompts_provider.agent_types import SystemAgentType
+
+      
+class BasePromptsProvider(ABC):
```

Ruff (0.8.2)

`BasePromptsProvider` is an abstract base class, but it has no abstract methods or properties

(B024)